home/jobs/OMERO-web/config.xml

```diff
@@ -181,6 +179,32 @@ pip install omero-webtagging-tagsearch
 omero config append omero.web.apps &apos;&quot;omero_webtagging_tagsearch&quot;&apos;
 omero config append omero.web.ui.top_links &apos;[&quot;Tag Search&quot;, &quot;tagsearch&quot;]&apos;
 
+# omero-web-zarr
+pip install git+https://github.com/$SPACE_USER/omero-web-zarr.git@$MERGE_PUSH_BRANCH#egg=omero-web-zarr
+omero config append omero.web.apps &apos;&quot;omero_web_zarr&quot;&apos;
+
+# OMERO.signup installation
+pip install git+https://github.com/$SPACE_USER/omero-signup.git@$MERGE_PUSH_BRANCH#egg=omero-signup
+omero config append omero.web.apps &apos;&quot;omero_signup&quot;&apos;
+omero config set omero.web.signup.admin.user &quot;root&quot;
+omero config set omero.web.signup.admin.password &quot;omero&quot;
+omero config set omero.web.signup.group.name &quot;signup-test&quot;
+omero config set omero.web.signup.email.enabled True
+
+# virtual microscope
+pip install git+https://github.com/$SPACE_USER/omero-virtual-microscope.git@$MERGE_PUSH_BRANCH#egg=omero-virtual-microscope
+omero config append omero.web.apps &apos;&quot;virtualmicroscope&quot;&apos;
+omero config set omero.web.login_redirect &apos;{&quot;redirect&quot;: [&quot;webindex&quot;], &quot;viewname&quot;: &quot;webindex_custom&quot;}&apos;
+
+
+# CORS
+pip install &quot;django-cors-headers==3.2.1&quot;  # 3.7.0 for Django 3.2
```

Comment:
That's effectively a downgrade from https://github.com/ome/omero-web/blob/59ec465df1a4f55c7097696647e14dccc810a602/setup.py#L59. Is this still a requirement?

Reply:
The PR was opened a while back. So that needs to be adjusted

Reply:
see [fdb265e](https://github.com/ome/devspace/pull/192/commits/fdb265ec85f2cc24f97cac8e2e19af28fcac2e95)